Swift/SelectResult.swift

```diff
@@ -111,7 +111,7 @@ public final class SelectResult {
 /* Internal */ class QuerySelectResultFrom: QuerySelectResult, SelectResultFrom {
     
     public func from(_ alias: String?) -> SelectResultProtocol {
-        return QuerySelectResult(expression: Expression.all().from(alias), alias: nil)
+        return QuerySelectResult(expression: Expression.all().from(alias), alias: alias)
```

Comment:
if we revert this change, the test will in Swift. Since the [SelectResult asJSON] is returning without alias. 

Query JSON without alias:
```
{"WHAT":[[".main."],[".secondary."]],"DISTINCT":true,"FROM":[{"AS":"main"},{"ON":["=",[".main.number1"],[".secondary.theOne"]],"AS":"secondary","JOIN":"INNER"}]}
```